osu.Game/Screens/Select/PlaySongSelect.cs

```diff
@@ -40,8 +46,10 @@ public PlaySongSelect()
         }
 
         [BackgroundDependencyLoader]
-        private void load(OsuColour colours)
+        private void load(OsuColour colours, UserInputManager input)
         {
+            this.input = input;

```

Comment:
really don't like this. handle keyboard KeyDown and KeyUp events and store the current state of control if anything.